<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>453</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sitka Heading</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scroll_area">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>571</width>
     <height>341</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="qwidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>552</width>
      <height>339</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Heading</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Поиск по названию</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Subheading</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Искать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="title_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Heading</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
